src/main/java/homer/common/DeviceTypesImpl.java

```diff
@@ -0,0 +1,40 @@
+package homer.common;
+
+import java.util.LinkedHashSet;
+import java.util.List;
+import java.util.Set;
+
+
+/**
+    Pre-populated valid device types.
+ */
+public final class DeviceTypesImpl implements DeviceTypes {
+
+    private final Set<String> validTypes;
+    private final Set<String> startingTypes = new LinkedHashSet<>(
+        List.of("LIGHT", "OUTLET", "COUTLET", "LOUTLET", "A/C", "HEATER")
+    );
```

Comment:
How are we supposed to retrieve the types from a collection when we want to use it?

Reply:
@aleemont1 @michelececcacci maybe we should scrap all of this and just use retrieve the class names via reflection if we need to display them.
Same if we want to use configuration files, we will use the class names to identify the devices.

Reply:
> leave DeviceInfo with just the id in there.
Feels like the better choice. By the way, how do we plan on assigning the id to the devices?

Reply:
Tbh I would just create the ID inside the DeviceInfoImpl, we don't have to edit or do anything to it.

Reply:
wait, i actually meant `scrap DeviceInfo too and just keep the id` feels like the best option. sorry for the confusion